docs/src/howto/install/multipass.md

```diff
@@ -1 +1,121 @@
 # Install with Multipass (Ubuntu/Mac/Windows)
+
+**Multipass** <https://multipass.run/> is a simple way to run Ubuntu in a
+virtual machine, no matter what your underlying OS. It is the recommended way
+to run Canonical Kubernetes on Windows and macOS systems, and is equally useful
+for running multiple instances of the `k8s` snap on Ubuntu too.
+
+## Install Multipass
+
+Choose your OS for the install procedure
+
+````{tabs}
+
+```{group-tab} Ubuntu/Linux
+
+Multipass is shipped as a snap for Ubuntu and other OSes which support the 
+[snap package system][snap-support].
+
+    sudo snap install multipass
+
+
+```
+
+```{group-tab} Windows
+
+Windows users should download and install the Multipass installer from the
+website. 
+
+The latest version is available here <https://multipass.run/download/windows>,
+though you may wish to visit the [Multipass website][] for more details.
+
+
+```
+
+```{group-tab} macOS
+
+Users running macOS should download and install the Multipass installer from the
+website. 
+
+The latest version is available here <https://multipass.run/download/macos>,
+though you may wish to visit the [Multipass website][] for more details, including
+an alternate install method using `brew`.
+
+```
+
+````
+
+## Create an instance
+
+The `k8s` snap will require a certain amount of resources, so the default
+settings for a Multipass VM aren't going to be suitable. Exactly what resources
+will be required depends on your use case. We recommend at least 4G of memory
+and 20G of disk space for each instance.
+
+Open a terminal (or Shell on Windows) and enter the following command:
+
+```no-highlight
+multipass launch 22.04 --name k8s-node -m 4G -d 20G -c 2
+```
+
+This command specifies:
+
+ - **22.04**: The Ubuntu image used as the basis for the instance 
+ - **--name**: The name by which you will refer to the instance
+ - **-m**: The memory to allocate
```

Comment:
can we use the long option names instead?

Reply:
yes, i did think that myself. it makes the example command very long, but you are undoubtedly correct